libraries/AP_Frsky_Telem/AP_Frsky_SPort_Passthrough.h

```diff
@@ -1,6 +1,9 @@
 #pragma once
 
 #include "AP_Frsky_SPort.h"
+
+#if AP_FRSKY_SPORT_PASSTHROUGH_ENABLED
```

Comment:
Somehow you need to be able to leave in the bits that CRSF passthrough needs unless both are disabled

Reply:
Yep, if you don't have passthrough then you don't get crsf.

I don't think that's a good situation, but factoring the code to avoid that is really beyond the scope here.

Reply:
This is the error you get if you try crsf without frsky:
```
../../libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp: In member function 'void AP_CRSF_Telem::setup_custom_telemetry()':
../../libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp:120:5: error: 'AP_Frsky_SPort_Passthrough' was not declared in this scope
  120 |     AP_Frsky_SPort_Passthrough* passthrough = AP::frsky_passthrough_telem();
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
```

This isn't something new.  It's a dependency which should be removed - but I'm not adding the dependency, just allowing it to trigger if someone tries to have crsf without frsky.